Add Axiom to Final Energy Consumption Value #1832

nelekoehler opened the pull request:
## Summary of the discussion

As discussed in #1832 I added an axiom to final energy consumption and since it is a quantity value I adjusted the label and the definition, so that the distinction between quantity and quantity value becomes more clear.

### Add
- Added an axiom: Equivalent to 'energy consuption value' and 'is about' some (consumption and 'has participant' some 'final energy carrier')

### Update
- changed label: final energy consumption
- changed definition: Final energy consumption value is an energy consumption value expressing the magnitude of the energy delivered to and consumed by end users.

## Workflow checklist

### Automation
Closes #1832

### PR-Assignee
- [x ] 🐙 Follow the [Pull Request Workflow](https://github.com/OpenEnergyPlatform/ontology/wiki/Pull-request-workflow)
- [x] 📝 Update the [CHANGELOG.md](https://github.com/OpenEnergyPlatform/ontology/blob/dev/CHANGELOG.md)
- [x] 📙 Add #'s to `term tracker item`

### Reviewer
- [ ] 🐙 Follow the [Reviewer Guide](https://github.com/OpenEnergyPlatform/ontology/wiki/Pull-request-workflow#reviewer-guide-check-changes-introduced-by-a-pull-request)
- [ ] 🐙 Provided feedback and show sufficient appreciation for the work done

stap-m commented:
I just saw that energy consumption value is not related to `consumption` or `energy use`. I'd proposed to add that axiom and have it inherited by `final energy consumption value`, instead of adding it to the equivalent axiom. Do you agree?

nelekoehler commented:
> I just saw that energy consumption value is not related to `consumption` or `energy use`. I'd proposed to add that axiom and have it inherited by `final energy consumption value`, instead of adding it to the equivalent axiom. Do you agree?

Yes I agree, we should add the axiom to energy consumption value. I will do that and adjust the axiom of final energy consumption value then, so that it inherits it. Should I add both 'energy use' and 'consumption' or only 'energy use' because it is a subclass of 'consumption'?

stap-m commented on CHANGELOG.md:
```diff
@@ -12,6 +12,7 @@ and the versioning aims to respect [Semantic Versioning](http://semver.org/spec/
 
 ### Changed
 - has documentation quality (#1834)
+- Final Energy Consumption Value, Energy Consumption Value (#1841)
```
```suggestion
- final energy consumption value, energy consumption value (#1841)
```

l-emele commented on src/ontology/edits/oeo-physical.omn:
```diff
@@ -10664,14 +10664,22 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/608",
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Final energy consumption is an energy consumption value accounting for the energy delivered to and consumed by end users."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Final energy consumption value is an energy consumption value expressing the magnitude of the energy delivered to and consumed by end users."@en,
```
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A final energy consumption value is an energy consumption value expressing the magnitude of the energy delivered to and consumed by end users."@en,
```

l-emele replied:
Please keep in mind, that we aligned the definitions of final energy consumption (value) and primary consumption (value). So if we change the definition of final energy consumption (value) then we should adapt also primary energy consumption (value).

Also we should keep `final energy consumption` at least as alternative term as this is the term that is used in the domain!

stap-m replied:
> Please keep in mind, that we aligned the definitions of final energy consumption (value) and primary consumption (value). So if we change the definition of final energy consumption (value) then we should adapt also primary energy consumption (value).

I agree. 
In this context, we should also fix the axiom of `primary energy consumption (value)`. Since it is a quantity value, I think we should avoid "has part". Instead: `not (is about some non-energy use)`. The other part will be inherited fom energy consumption value.

l-emele replied:
I am fine with the proposal for the axiomatic change of primary energy consumption.

nelekoehler commented:
I've made the axiomatic change of primary energy consumption and changed the label. Since we were already adjusting the consumptions value, I changed the labels of the other to consumption values, added the old label as an alternative label and changed the axiom of gross inland energy consumption also to 'is about' some 'non-energy-use' instead of 'has part'. Do you agree with these changes?

l-emele commented on src/ontology/edits/oeo-physical.omn:
```diff
@@ -10707,21 +10716,20 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/613
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Primary energy consumption is an energy consumption value accounting for the total consumption of energy in a spatial region excluding the non-energetic use of fuels."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Primary energy consumption value is an energy consumption value accounting for the total consumption of energy in a spatial region excluding the non-energetic use of fuels."@en,
```
Align with `final energy consumption value`:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A primary energy consumption value is an energy consumption value expressing the magnitude of the total consumption of energy in a spatial region excluding the non-energetic use of fuels."@en,
```

l-emele commented on src/ontology/edits/oeo-physical.omn:
```diff
@@ -10680,7 +10689,7 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1340",
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Gross inland energy consumption is an energy consumption value accounting for the total consumption of energy in a spatial region (e.g. a country)."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Gross inland energy consumption value is an energy consumption value accounting for the total consumption of energy in a spatial region (e.g. a country)."@en,
```
Align with final energy consumption value:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A gross inland energy consumption value is an energy consumption value expressing the magnitude of the total consumption of energy in a spatial region (e.g. a country)."@en,
```

stap-m commented:
Could you please add an axiom to `primary energy carrier` to `primary energy consumption value`. To harmonize it with `final energy consumption value`, also make it a defined class?!

l-emele commented:
> Could you please add an axiom to `primary energy carrier` to `primary energy consumption value`.

Why? Primary energy consumption is **not** the consumption of primary energy carriers.

l-emele commented on CHANGELOG.md:
```diff
@@ -12,6 +12,7 @@ and the versioning aims to respect [Semantic Versioning](http://semver.org/spec/
 
 ### Changed
 - has documentation quality (#1834)
+- final energy consumption value, energy consumption value (#1841)
```
Mention all change classes and superclass first:
```suggestion
- energy consumption value, final energy consumption value, primary energy consumption value, gross inland energy consumption value, gross national electricity consumption value (#1841)
```

l-emele reviewed (commented):
I am a bit unhappy about the axiomatisation of `primary energy consumption value` and `gross inland energy consumption value`. The axioms basically say, that any energy consumption value which excludes non-energy use is a PEC value and any energy consumption value which included non-energy use is a GIEC value. But this is not true.

nelekoehler commented:
> I am a bit unhappy about the axiomatisation of `primary energy consumption value` and `gross inland energy consumption value`. The axioms basically say, that any energy consumption value which excludes non-energy use is a PEC value and any energy consumption value which included non-energy use is a GIEC value. But this is not true.

Do you have an idea how to change that? If not, I will look into it next week and try to make a new suggestion.

nelekoehler commented:
I will change the axioms about the non-energy use of final energy consumption value and gross inland consumption value from EquivalentTo to SubclassOf.

nelekoehler commented:
@l-emele could you review this PR again? The Merging is still blocked, because you requested changes, which I have already implemented now.
